etc/ci/upload_nightly.sh

```diff
@@ -22,17 +23,21 @@ main() {
         return 1
     fi
 
-    local platform package
+    local platform package ext
     platform="${1}"
 
     if [[ "${platform}" == "android" ]]; then
         package=target/arm-linux-androideabi/release/*.apk
+        ext=".apk"
```

Comment:
Put the `ext=` lines above the `package=` lines, and use the `ext` variable to keep the extension in sync (i.e. when we have .msi for windows).